### PR TITLE
Use tools.mk in Makefile

### PR DESCRIPTION
**Pull request by Vagabond**
We intentionally let the Makefile override the 'test' target from
tools.mk, because it does weird stuff.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -26,35 +31,3 @@ endif
 
 test: all test-compile
```
Comment (seancribbs) [inline Makefile]: Is it possible to remove the warning by simply adding test-compile as a dependency but remove the command?

Comment (seancribbs) [inline Makefile]: Yep, this seems to work:

``` make
test: test-compile
```

Comment (Vagabond) [inline Makefile]: I don't understand what that means. Is that adding a dep to an existing target?

Comment (seancribbs) [inline Makefile]: Yes.

**Comment by seancribbs** [comment]
Other than fixing the `test` target, :+1:

**Comment by seancribbs** [comment]
:+1: again

**Pull request merged**